api/views.py

```diff
@@ -129,9 +131,10 @@ def delivery_log_webhook(request, id):
     # Write audit log of the action
     audit_log_service._commit_to_audit_log(
         message=create_api_commit_message_from_request(
-            request,
-            Operation.UPDATE.value,
-            [str(id)],
+            request=request,
+            operation=Operation.UPDATE.value,
+            object_ids=[str(id)],
+            old_objects=[log],
```

Comment:
Isn't the `log` here actually a new object? I mean `log.update_report(request.data)` above has already been run. Having the old object also would be nice as this is an update operation. I'm thinking how'd that be possible, a deep-copy perhaps?

Reply:
changed to `new_objects`